### PR TITLE
fix compile assets path error

### PR DESCRIPTION
**Pull request by tim0991**

### CODE REVIEW

File: modules/backend/assets/js/october.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 =require ../vendor/sweet-alert/sweet-alert.js
 =require ../vendor/jcrop/js/jquery.Jcrop.js
 =require ../../../system/assets/vendor/prettify/prettify.js
-=require ../../../cms/widgets/mediamanager/assets/js/mediamanager-global.js
+=require ../../backend/widgets/mediamanager/assets/js/mediamanager-global.js
```
Comment (LukeTowers) [inline modules/backend/assets/js/october.js]: Good catch! However, wouldn't this only require `../widgets`?, not `../../backend/widgets`? @tim0991

Comment (LukeTowers) [reply modules/backend/assets/js/october.js]: Because it's already in the backend?

Comment (tim0991) [reply modules/backend/assets/js/october.js]: It's my fault,  i will push a new pr 😂

Comment (LukeTowers) [reply modules/backend/assets/js/october.js]: @tim0991 no need to push a new PR, you could have just added a commit to the branch that you made these changes on

**Pull request closed without merging**